hydroflow/src/util/mod.rs

```diff
@@ -32,6 +32,17 @@ pub fn unbounded_channel<T>() -> (
     (send, recv)
 }
 
+pub fn bounded_channel<T>(
+    capacity: usize,
+) -> (
+    tokio::sync::mpsc::Sender<T>,
+    tokio_stream::wrappers::ReceiverStream<T>,
+) {
+    let (send, recv) = tokio::sync::mpsc::channel(capacity);
+    let recv = tokio_stream::wrappers::ReceiverStream::new(recv);
+    (send, recv)
+}
+
```

Comment:
Let's remove this if its unused since it is a bit of a footgun with the current `dest_sink` impl